### PR TITLE
Minor code changes

### PR DESCRIPTION
**Pull request by kshinde2512**
### Describe your changes :

I worked on the minor code change in without ingestion tests

#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@darth-coder00 @shahsank3t

### CODE REVIEW

**Review by harshach — APPROVED** [review]

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java [hunk 1]
```diff
@@ -80,7 +81,42 @@
   By topics = By.xpath("(//button[@data-testid='tab'])[2]");
   By selectTable = By.xpath("(//button[@data-testid=\"table-link\"])[2]");
   By addTag = By.xpath("//div[@data-testid='tag-conatiner']");
-  By breadCrumbTag = By.xpath("(//span[@data-testid='tags'])[1]");
+  By breadCrumbTags = By.xpath("(//span[@data-testid='tags'])[1]");
+  By ingestionTab = By.xpath("//button[@data-testid='tab'][@id='ingestions']");
+  By addIngestion = By.cssSelector("button[data-testid='add-new-ingestion-button']");
+  By name = By.cssSelector("[data-testid='name']");
+
+  public By name() {
```
Comment (mithmatt) [inline openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java:89]: The methods to get these fields are no longer needed.

They get auto-generated by the "@Getter" annotation above.

**Review by mithmatt — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java (hunk 1 above)
Comment (mithmatt) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java:89]: See #2975

Comment (kshinde2512) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java]: Removed the methods that were no longer needed

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 @RequiredArgsConstructor
 public class Common {
   @Nonnull WebDriver webDriver;
```
Comment (mithmatt) [inline openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Common.java:14]: Should probably change this to lombok.NonNull as well

**Review by mithmatt — APPROVED** [review]

**Comment by harshach** [comment]
@kshinde2512 can you rebase this PR

**Pull request closed without merging**